packages/core/admin/server/services/permission/permissions-manager/sanitize.js

```diff
@@ -263,6 +263,8 @@ module.exports = ({ action, ability, model }) => {
       CREATED_AT_ATTRIBUTE,
       UPDATED_AT_ATTRIBUTE,
       PUBLISHED_AT_ATTRIBUTE,
+      CREATED_BY_ATTRIBUTE,
+      UPDATED_BY_ATTRIBUTE,
```

Comment:
For non-visible fields, they need to be added to RBAC but we don't currently have that on the roadmap -- but it's not very complex so someone may end up doing it on the side, or anyone is welcome to.

The only way I can envision it is adding all non-visible fields (or aggregate them into "creator fields" and "review worklows" and anything else I might be forgetting) to RBAC. So it would be added either as part of fields permissions with only a "read" checkbox, or under another header like "Non-visible fields":

<img width="852" alt="image" src="https://github.com/strapi/strapi/assets/999278/8eb181cd-2246-4c3a-8f3b-9250015c70ce">

If anyone wants to take this on, just double-check with @marcoautiero on how it should work (like if we only allow "read" since updating